src/source/restful/kvs/restapi_kvs.c

```diff
@@ -705,83 +720,87 @@ int Kvs_describeStream(KvsServiceParameter_t *pServPara, KvsDescribeStreamParame
         LogError("Invalid argument");
         /* Propagate the res error */
     }
-    else if ((res = getTimeInIso8601(pcXAmzDate, sizeof(pcXAmzDate))) != KVS_ERRNO_NONE)

```

Comment:
Note for reviewers - this section, I wanted to add these two log lines:

```
LogInfo("Making describe request to: %s", pServPara->pcHost);
LogInfo("Stream name: %s", pDescPara->pcStreamName);
```

But, LogInfo doesn't return anything, so I couldn't put it in the condition. Therefore, I moved it to the `else` and moved the rest of the conditions into the `else` as well (indent).